test/logger.test.js

```diff
@@ -13,31 +13,19 @@ const fs = require('fs')
 const sget = require('simple-get').concat
 const dns = require('dns')
 
-const files = []
 let count = 0
 let localhost
 let localhostForURL
 
 function file () {
   const file = path.join(os.tmpdir(), `sonic-boom-${process.pid}-${process.hrtime().toString()}-${count++}`)
-  files.push(file)
   return file
 }
 
 before(async function () {
   [localhost, localhostForURL] = await helper.getLoopbackHost()
 })
 
-teardown(() => {
```

Comment:
Should we only omit the teardown in CI? This is going to leave behind trash on non-ephemeral systems.